src/main/resources/assets/metamorph/lang/zh_CN.lang

```diff
@@ -4,90 +4,171 @@ entity.metamorph.Morph.name=Morph
+metamorph.gui.acquired=已获取
 metamorph.gui.close=关闭
-metamorph.gui.remove=移除
-metamorph.gui.favorite=喜欢
-metamorph.gui.unfavorite=取消喜欢
 metamorph.gui.search=搜索
-metamorph.gui.only_favorites=展示喜欢的
-metamorph.gui.all_morphs=展示全部
-metamorph.gui.no_morph=没有选择伪装
-metamorph.gui.selected=选择
+metamorph.gui.no_morph=没有选中伪装...
+metamorph.gui.morph_render_error=渲染该伪装时发生错误
+
+metamorph.gui.selectors.title=实体选择器
+metamorph.gui.selectors.tooltip=通过此功能, 你可以为指定名称或指定类型的实体添加伪装...
+metamorph.gui.selectors.add=添加选择器
+metamorph.gui.selectors.remove=移除选择器
+metamorph.gui.selectors.name=名称
+metamorph.gui.selectors.type=类型
+metamorph.gui.selectors.match=匹配 NBT
+metamorph.gui.selectors.match_tooltip=你可以在此处指定一个 NBT 标签, 该标签必须与实体的 NBT 标签部分匹配
+metamorph.gui.selectors.enabled=启用
+
+metamorph.gui.morphs.keys.category=伪装列表导航
+metamorph.gui.morphs.keys.up=向上选择伪装
+metamorph.gui.morphs.keys.down=向下选择伪装
+metamorph.gui.morphs.keys.right=向右选择伪装
+metamorph.gui.morphs.keys.left=向左选择伪装
+
+metamorph.gui.creative.command=为所选的伪装复制 /morph 命令
+metamorph.gui.creative.edit=编辑
+metamorph.gui.creative.pick=选取
+metamorph.gui.creative.quick=快速编辑
+metamorph.gui.creative.presets=预设
+metamorph.gui.creative.random=随机预设
+metamorph.gui.creative.keys.category=创造模式伪装菜单
+metamorph.gui.creative.keys.exit=退出伪装编辑器
+metamorph.gui.creative.keys.edit=编辑当前所选的伪装
+metamorph.gui.creative.keys.focus=聚焦至搜索框
+metamorph.gui.creative.keys.quick=调出快速编辑菜单
+metamorph.gui.creative.keys.acquire=获取当前所选的伪装
+metamorph.gui.creative.keys.morph=伪装成当前所选的伪装
+metamorph.gui.creative.context.add_global=添加为全局伪装...
+metamorph.gui.creative.context.edit=编辑伪装...
+metamorph.gui.creative.context.add_category=添加新类别
+metamorph.gui.creative.context.rename_category=重命名类别
+metamorph.gui.creative.context.rename_category_modal=为该类别命名...
+metamorph.gui.creative.context.remove_category=移除分类
+metamorph.gui.creative.context.clear_category=清空
+metamorph.gui.creative.context.to_recent=重复到最近的伪装
+metamorph.gui.creative.context.remove_morph=移除伪装
+metamorph.gui.creative.context.copy=重复 NBT
+metamorph.gui.creative.context.paste=粘贴 NBT
+metamorph.gui.creative.context.paste_modal=粘贴或输入你伪装的 NBT 标签...
+
+metamorph.gui.survival.title=生存模式伪装菜单
+metamorph.gui.survival.keybind_tooltip=你可以通过此输入框绑定一个快捷键, 这个快捷键会让你变到这个伪装 (当你在菜单或游戏世界中按下时触发)
+metamorph.gui.survival.only_favorites=仅显示收藏
+metamorph.gui.survival.favorite=收藏
+metamorph.gui.survival.keys.category=生存模式伪装菜单
+metamorph.gui.survival.keys.toggle_favorites=将伪装切换为收藏
+metamorph.gui.survival.keys.focus_keybind=聚焦快捷键
+
+metamorph.gui.editor.settings=伪装设置
+metamorph.gui.editor.display_name=显示名称
+metamorph.gui.editor.health=生命值
+metamorph.gui.editor.speed=速度
+metamorph.gui.editor.abilities=技能
+metamorph.gui.editor.abilities_tooltip=按住 Shift 并单击列表中的技能, 即可进行多选
+metamorph.gui.editor.attack=攻击
+metamorph.gui.editor.action=动作
+metamorph.gui.editor.reset=重置
+metamorph.gui.editor.item_morph=物品
+metamorph.gui.editor.keys.category=Morph editor keybinds
+metamorph.gui.editor.keys.cycle=Cycle between morph panels
```

Comment:
Did you forget to translate these strings? (same for zh-TW as well)

Reply:
Sorry for this omission, the translation has now been completed. Thank you for pointing out this problem!